onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -120,7 +120,6 @@ def quantize(model,
     else:
```

Comment:
no meaningful modifications to this file?

Reply:
Yes. This PR doesn't modify quantize.py